rec/.htaccess

```diff
@@ -5,26 +5,8 @@ RewriteEngine on
 # Redirect any module w/o version in IRI to latest version (at time of writing 2.2)
 RewriteRule ^(\w+?)/(\w*?)$ http://w3id.org/rec/$1/2.2/$2 [R=303,NE,L]
```

Comment:
While you're changing things, can you switch this to https to avoid an extra auto-redirect?

Reply:
Thanks, good catch.